src/components/SimpleTable.tsx

```diff
@@ -125,49 +31,111 @@ export interface IProps {
-            {image && <StyledImage src={image} alt="" />}
-            {name}
-          </Cell>
-          <Cell>{marketCap}</Cell>
-          <Cell>{type}</Cell>
-        </>
-      )
+const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
+  const [textColor] = useToken("colors", ["text"])
+
+  return (
+    <Table
+      variant="unstyled"
+      my={8}
+      borderRadius="sm"
+      border={`1px solid ${textColor}`}
+      bg="background"
+      mb={8}
+      minW="720px"
+    >
+      <Thead bg="ednBackground" color="text200">
+        <Tr boxShadow={`0 1px 1px ${textColor}`} mb="1px">
+          {columns.map((column, idx) => (
+            <Th
+              key={idx}
+              py={5}
+              fontSize="md"
+              fontWeight="normal"
+              letterSpacing="normal"
+            >
+              {column}
+            </Th>
+          ))}
 
-      return url ? (
-        <LinkRow key={idx} to={url}>
-          {rowContent}
-        </LinkRow>
-      ) : (
-        <Row key={idx}>{rowContent}</Row>
-      )
-    })}
-  </Table>
-)
+          {content && content[0]?.url && (
+            <Th p={5} fontSize="md" fontWeight="normal" textAlign="end">
+              ↗
+            </Th>
+          )}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {hasError && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-error" />
+            </Td>
+          </Tr>
+        )}
+        {!hasError && content.length === 0 && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-loading" />
+            </Td>
+          </Tr>
+        )}
 
+        {content.map(({ name, marketCap, image, type, url }, idx) => (
+          <LinkBox
+            as={Tr}
+            display="table-row"
+            href={url}
+            key={idx}
+            color="text"
+            textDecoration="none"
+            boxShadow={`0 1px 1px ${textColor}`}
+            _hover={{
+              textDecoration: "none",
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+              cursor: url ? "pointer" : "default",
+            }}
+            _focus={{
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+            }}
+          >
+            <Td>
+              <Flex align='center'>
+                {image && <Image src={image} alt="" mr={4} boxSize={6} />}
+                {url ? (
+                  <LinkOverlay
+                    as={Link}
+                    hideArrow
+                    to={url}
+                    color="inherit"
+                    textDecoration="inherit"
+                    _hover={{
+                      textDecoration: "inherit",
+                    }}
+                    isExternal
+                  >
+                    {name}
```

Comment:
This is a personal recommendation, as it is a minor addition for A11y. (For @pettinarip approval 😄 )

Include visually hidden text for the `LinkOverlay` render only which reads `(Opens a new window)` (with the parens) since all the link that are ever passed into this table are assumed to be external.

> Maybe should have this under a translation function too and supplied in the `intl/en/common.json` file
```suggestion
                    {name}
                    <VisuallyHidden>
                      <Translation id="sr-link-opens-new-window" />
                    </VisuallyHidden>
```

And the json add:
`"sr-link-opens-new-window": "(Opens a new window)"`

Reply:
Hey @TylerAPfledderer, should I make these changes or wait for @pettinarip approval?

Reply:
@amit-ksh I would wait to make sure it's OK since it is adding to the structure.

Reply:
I think this is a worthy discussion we can have with the rest of the team on a different issue since I think that if we apply that change we should do the same with the rest of the external links we have on the site, to keep consistency. For that reason, I don't see that change as part of the scope of this PR/migration.

Happy to see this issue created and discussed with more people. I haven't seen this pattern a lot on other sites. I'm curious to hear what others think about it.